terraform/deploy/modules.tf

```diff
@@ -61,6 +61,7 @@ module "concourse_web" {
   cognito_domain        = module.cognito.outputs.user_pool_domain
   cognito_issuer        = module.cognito.outputs.issuer
   cognito_name          = module.cognito.outputs.name
+  remote_state          = local.remote_state
```

Comment:
I don't think this is needed anymore? I can't see the vars/locals referenced in any resources, unless I'm missing something?